pkg/kubectl/cmd/apply/apply.go

```diff
@@ -812,6 +836,13 @@ func (p *Patcher) patchSimple(obj runtime.Object, modified []byte, source, names
 		return patch, obj, nil
 	}
 
+	if p.ResourceVersion != nil {
```

Comment:
Won't patchSimple will fail O(retries) times if `p.ResourceVersion != nil` and you get a conflict? Should you do this in Patch() before the first patchSimple and not enter the retry loop if `p.ResourceVersion != nil`?